### PR TITLE
Cortx 33520 test

### PR DESCRIPTION
**Pull request by pragamj**
# Problem Statement
- Test TEST-41912,TEST-42913

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [x] Collection tested and no collection error introduced (`pytest --local True --collect-only`)
[collections_log.txt](https://github.com/Seagate/cortx-test/files/9441246/collections_log.txt)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: comptests/motr/test_motr_di.py [hunk 1]
```diff
@@ -289,28 +261,23 @@ def m0cat_md5sum_m0unlink(self, bsize_list, count_list, layout_ids, object_list,
         Validate the corruption with md5sum after M0CAT and unlink the object
         """
         logger.info("STARTED: m0cat_md5sum_m0unlink workflow")
-        infile = kwargs.get("infile", TEMP_PATH + "input")
-        outfile = kwargs.get("outfile", TEMP_PATH + "output")
+        infile = kwargs.get("infile", TEMP_PATH + 'input')
```
Comment (pranav-sahasrabudhe) [inline comptests/motr/test_motr_di.py:264]:
```suggestion
        infile = kwargs.get("infile", TEMP_PATH + "input")
```

File: comptests/motr/test_motr_di.py [hunk 2]
```diff
@@ -289,28 +261,23 @@ def m0cat_md5sum_m0unlink(self, bsize_list, count_list, layout_ids, object_list,
...
+        outfile = kwargs.get("outfile", TEMP_PATH + 'output')
```
Comment (pranav-sahasrabudhe) [inline comptests/motr/test_motr_di.py:265]:
```suggestion
        outfile = kwargs.get("outfile", TEMP_PATH + "output")
```

**Review by pranav-sahasrabudhe — CHANGES_REQUESTED** [review]
Minor changes

**Review by ketanarlulkar — APPROVED** [review]

**Review by ravindrachoudhari — APPROVED** [review]

File: commons/commands.py [hunk 3]
```diff
@@ -401,7 +401,6 @@
 
 M0CP_G = "m0cp -G -l $ep -H $hax_ep -P $fid -p $prof_fid -s $bsize -c $count -o $obj -L" \
          " $layout $file"
-
 M0CP_U = "m0cp -G -l $ep -H $hax_ep -P $fid -p $prof_fid -s $bsize -c $count -o $obj -L" \
```
Comment (pranav-sahasrabudhe) [inline commons/commands.py:404]:
```suggestion
M0CP_G_U = "m0cp -G -l $ep -H $hax_ep -P $fid -p $prof_fid -s $bsize -c $count -o $obj -L" \
```

**Review by pranav-sahasrabudhe — APPROVED** [review]

**Pull request merged**